README.md

```diff
@@ -1,36 +1,83 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+<img src = "https://github.com/fiizzy/calentre/blob/main/doc/images/logo.png?raw=true"  >
 
-## Getting Started
+# Calentre 📅💰
 
-First, run the development server:
+Welcome to Calentre, the open-source alternative to Calendly! 🌟 Manage your appointments, get paid, and enjoy the scheduling experience.
 
-```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-# or
-bun dev
-```
+## Table of Contents 📚
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+- [Description](#description)
+- [Features](#features)
+- [Technology Stack](#technology-stack)
+- [Architecture](#architecture)
+- [Getting Started](#getting-started)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
 
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+## Description 📝
 
-This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
+Calentre is your go-to scheduling solution, offering a delightful way to book meetings and collect payments! 🚀 Whether you're a consultant, freelancer, or professional, we've got you covered.
 
-## Learn More
+<img src = "https://github.com/fiizzy/calentre/blob/main/doc/images/home_events.png?raw=true" >
 
-To learn more about Next.js, take a look at the following resources:
+---
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+<img src = "https://github.com/fiizzy/calentre/blob/main/doc/images/booking.png?raw=true" >
```

Comment:
Let's take out this image, and replace it with Client image. A screenshot can be taken from your dev app.